mode-apis-base/src/alertingModels.ts

```diff
@@ -163,12 +163,47 @@ export const isMetricsThresholdAlertCondition = (obj: unknown): obj is MetricsTh
 
 
 
-/**
- * TODO - Define the attributes required for AlertAction
- */
-export interface AlertAction {
+export interface AlertMessageTemplate {
+    readonly title: string;
+    readonly message: string;
+    readonly recoveryMessage?: string;
+}
+
+export type AlertRecipientType = 'email';
```

Comment:
Can you make this enum? It would be easier to loop through enums.
```suggestion
export enum AlertRecipientType {
    EMAIL = 'email',
}
```

Reply:
Ah, that makes sense. I will change it.